Library/DiffinDiff_MHE.Rmd

```diff
@@ -24,38 +24,56 @@ pacman::p_load(packages, character.only = TRUE, install = FALSE)
 # Autor (2003): "Outsourcing at Will: The Contribution of Unjust Dismissal Doctrine to the Growth of Employment Outsourcing"
 data <- read_dta("https://github.com/worldbank/r-econ-visual-library/raw/master/Library/Data/autor-jole-2003.dta")
 
+# Printing the first five rows of the data to visualize its structure before data wrangling
+head(data)
+
+# Data wrangling ---------------
+
 admico_list <- c("admico_2", "admico_1", "admico0", "admico1", "admico2", "admico3", "mico4")
 admppa_list <- c("admppa_2", "admppa_1", "admppa0", "admppa1", "admppa2", "admppa3", "mppa4")
 admgfa_list <- c("admgfa_2", "admgfa_1", "admgfa0", "admgfa1", "admgfa2", "admgfa3", "mgfa4")
-labels <- c("2 Years\nprior", "1 Year\nprior", "Year of\nadoption", "1 Year\nafter", "2 Years\nafter", "3 Years\nafter", "4+ Years\nafter")
+labels <- c("2 Years\nprior", "1 Year\nprior", "Year of\nadoption", "1 Year\nafter", "2 Years\nafter", "3 Years\nafter", "4+ Years\nafter") # using \n to create a new line (\ is an escape operator and \n means a new line)
```

Comment:
This line is getting very long, so adding line breaks between arguments would make it easier to read. The comment is great!

Reply:
though the part about the escape operator is true, I don't think it's relevant in this case because the `\` symbol is not being used as such, but as part of the whole new line character `\n`. If `\` was being used as an escape operator followed by a new line, we would actually see `\\n` in the code.
```suggestion
labels <- c("2 Years\nprior", "1 Year\nprior", "Year of\nadoption", "1 Year\nafter", "2 Years\nafter", "3 Years\nafter", "4+ Years\nafter") # using \n to create a new line (the \n character means a new line)
```
In any case, it's nice to explain the meaning of `\n`. It can be confusing the first time you see it 👍